src/public.ts

```diff
@@ -153,3 +154,39 @@ export function isReleaseCandidate(version: string) {
 export function isSnapshot(version: string) {
     return sls.isSnapshot(sls.parse(version));
 }
+
+/**
+ * Determine if a version matcher string is a valid SLS version matcher
+ *
+ * @param versionMatcher Version matcher string to check
+ */
+export function safeValueOf(versionMatcher: string) {
+    return SlsVersionMatcher.safeValueOf(versionMatcher);
+}
+
+/**
+ * Determine if an SLS version matches a version matcher string
+ *
+ * @param version Version string to check
+ * @param versionMatcher Version matcher string to check against
+ */
+export function matches(version: string, versionMatcher: string) {
+    try {
+        const slsVersion = sls.parse(version);
+        const slsVersionMatcher = SlsVersionMatcher.parse(versionMatcher);
+        return slsVersionMatcher.matches(slsVersion);
+    } catch (error) {
+        return false;
+    }
+}
+
+/**
+ * @return a positive number if version > versionMatcher, negative number if version < versionMatcher, 0 if they match
+ * @param version
+ * @param versionMatcher
+ */
+export function compareToMatcher(version: string, versionMatcher: string) {
```

Comment:
same thing here. Want to keep the API of this package minimal

Reply:
sure, we'll need to expose `parse` from `slsVersion` in that case since `SlsVersionMatcher` expects an `ISLSVersion`